invenio_pages/config.py

```diff
@@ -8,6 +8,8 @@
 
 """Page module config."""
 
+from flask_babelex import lazy_gettext as _
```

Comment:
@utnapischtim having your changes in mind, is this import correct?

Reply:
@kpsherva Reminder to take this into account after releasing the changes.